.devcontainer/postAttachWelcomeText.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+clear
+
+echo "========================================="
+echo " ✅ Getting Started with the Stellar CLI"
+echo "========================================="
+echo ""
+echo "  ⚙️ Environment variables:"
+echo "  STELLAR_RPC_URL: ${STELLAR_RPC_URL}"
+echo "  STELLAR_NETWORK_PASSPHRASE: ${STELLAR_NETWORK_PASSPHRASE}"
+echo ""
+echo "  🆔 Configure an Identity:"
+echo "  stellar keys generate --global alice --network testnet --fund"
+echo "  stellar keys address alice"
+echo ""
+echo "  🧪 Run tests"
+echo "  cargo test"
```

Comment:
I think might be worth adding the build command too: `stellar contract build` before the run tests.